_includes/post.json

```diff
@@ -0,0 +1,8 @@
+{
+  "title"    : "{{ post.title }}",
+  "category" : "{{ post.category }}",
+  "tags"     : [{% for tag in post.tags %}"{{ tag }}",{% endfor %} null],
+  "url"     : "{{ post.url }}",
+  "title_image" : "{{ post.title_image }}",
+  "prep_time" : "{{ post.prep_time }}"
+}
```

Comment:
We might want to think about the best way to escape these properties - if someone creates a recipe with a title containing double quotes, this will break